lib/project_types/theme/messages/messages.rb

```diff
@@ -57,6 +57,7 @@ module Messages
               Usage: {{command:%s theme push [ ROOT ]}}
 
               Options:
+                {{command:-t, --theme=THEME}}     Theme ID or name of the remote theme.
                 {{command:-i, --themeid=THEMEID}} Theme ID. Must be an existing theme on your store.
```

Comment:
Any reason, other than backward compat, to keep this arg now that we have `-t`?

If it's just for backward compatibility, maybe we should deprecate `-i` and remove it from the usage doc?

Reply:
Excellent suggestion! Fixed :)